app/models/database.py

```diff
@@ -0,0 +1,57 @@
+from typing import Optional
+from boto3.dynamodb.conditions import Key
+from boto3.session import Session
+from pydantic import BaseModel
+
+
+#this file will contain connection to the database, and database logic
+
+
+TABLE_NAME = "LinkShortener"
```

Comment:
This should be better named. "LINKS_TABLE_NAME"

Reply:
Noted, will change this